src/biz/collab/ops.rs

```diff
@@ -293,10 +303,86 @@ pub async fn get_user_trash_folder_views(
   Ok(section_items_to_trash_folder_view(&section_items, &folder))
 }
 
+#[allow(clippy::too_many_arguments)]
+fn patch_old_workspace_folder(
+  user: RealtimeUser,
+  workspace_id: &str,
+  folder: &mut Folder,
+  child_view_id_without_space: &[String],
+) -> Result<Vec<u8>, AppError> {
+  let encoded_update = {
+    let space_id = Uuid::new_v4().to_string();
+    let view = NestedChildViewBuilder::new(user.uid, workspace_id.to_string())
```

Comment:
```rust
 pub fn space_info(&self) -> Option<SpaceInfo> {
    let extra = self.extra.as_ref()?;
    serde_json::from_str::<SpaceInfo>(extra).ok()
  }
```
in collab repo, we have a method call space_info(). So we need to insert this space info in the view extra

Reply:
I think you can use `space_info` instead of building the extra value manually.